src/external_app/external_events_forwarder.ts

```diff
@@ -0,0 +1,15 @@
+export const externalForwardConnectionEvents = (bus) => {
+  const sendConnEvent = (evtType: string) => {
+    bus.fireMessage({ type: "connection-status", result: { event: evtType } });
+  };
+
+  document.addEventListener("connected", () => {
+    sendConnEvent("connected");
+  });
+  document.addEventListener("disconnected", () => {
```

Comment:
We shouldn't need to add these, these should be sent as part of `connection-status` events inside the code too.